gfx/UltimateCataclysm/pngs_tall_32x64/monsters/mon_zombie_headless_variants/mon_zombie_headless.json

```diff
@@ -1,10 +1,15 @@
 [
-{
+  {
   "id": "mon_zombie_brainless",
     "fg": [
       { "weight": 100, "sprite": "zombie_headless_f" },
       { "weight": 100, "sprite": "zombie_headless_m" }
     ],
   "bg": "shadow_32x64_normal"
-}
+  },
+  {
+    "id": "corpse_mon_zombie_brainless",
+    "fg": "corpse_mon_zombie_brainless"
+  }
 ]
+    
```

Comment:
```suggestion
```
try to not add unecessary lines, you can the dda linter http://dev.narc.ro/cataclysm/format.html to format the json nicely